### ISSUE TITLE
Snaps folder: Question about Imagick for php 8.1

### ISSUE THREAD

**Issue by tmb-github** [Imagick/imagick#523]
I found a "snaps" folder with Imagick compiled against PHP 8.1, and I have successfully installed and used it in the current XAMPP with PHP 8.1.1 installed:

https://windows.php.net/downloads/pecl/snaps/imagick/3.6.0/

Are these Imagicks considered good to go, or are they awaiting additional testing before being put in the "releases" folder? What distinguishes the snaps?

**Comment by Danack** [comment]
> Are these Imagicks considered good to go,

Kind of.....but complicated.

The snaps are fine as far as Imagick itself is concerned, but they were built against a version of ImageMagick that appears to have a couple of bugs in it e.g. https://github.com/Imagick/imagick/issues/485 . If that bug doesn't affect you, i.e. if that release appears to work for you, then go for it.

CMB has kindly done quite a bit of work to automate upgrading the version of ImageMagick used to make the builds. My understanding is that the next release of Imagick will be against a new version of ImageMagick that doesn't have that issue. Though doing a release over christmas seems like a 'not great' idea.

> are they awaiting additional testing before being put in the "releases" folder?

So there's a notice on [windows.php.net](https://windows.php.net/)

> but you need one or more PECL packages for which there are no PHP 8.1 binary packages available yet, and you may wonder when these will be made available. Well, the PHP for Windows team does not want to provide possibly premature binary packages, so as usual, we will wait until late January or early February to trigger the mass build for all applicable PECL packages;

I presume this is mostly to avoid making packages available when the extension authors haven't full upgraded them and/or been tested. I've traditionally stayed away from the Windows builds as I don't use Windows and haven't been in a position to provide builds myself.

However, with the move to github actions, this means that I could provide Windows builds at some point in the future, which would be nice, as I could more easily control which version of ImageMagick is used, but that is for the future.

tl:dr if it works for you, then use it. If you get funny image results from it, wait till the next release which should be just after christmas.

**Comment by tmb-github** [comment]
@Danack Thanks! Now I have clarity. I'll report any odd results I get while waiting for the next release.

**Comment by Danack** [comment]
Cool. Closing as nothing else to do.
